core/Tracker/Db/Pdo/Mysql.php

```diff
@@ -339,14 +412,7 @@ private function establishConnection(): void
             $sql = "SET NAMES '".$this->charset."'";
             $this->connection->exec($sql);
         }
-
-        $this->resetPassword();
     }
 
-    private function resetPassword()
```

Comment:
the removal of this shouldn't cause any issues as in other db providers we never did that and it wasn't a problem